style.css

```
@@ -0,0 +1,146 @@
+header {
+    width: 100%;
+    background-color: #ffffff;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.24), 0px 0px 4px rgba(0, 0, 0, 0.12);
+}
+.header__content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 95%;
+    max-width: 1440px;
+    margin: 0 auto;
+    padding: 15px 0;
+
+    font-family: "Roboto";
+    font-size: 20px;
+    line-height: 23px;
+    color: rgba(0, 0, 0, 0.54);
+}
+.header__menu-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 220px;
+    position: relative;
+}
+.header__menu-header h1 {
+    font-weight: 500;
+    color: rgba(0, 0, 0, 0.87);
+}
+.header__menu-header .fa-bars:hover,
+.header__settings .fa-ellipsis-v:hover {
+    transform: scale(1.3);
+    color: #686767;
+    cursor: pointer;
+}
+
+body {
+    width: 100%;
+    background-color: #ffffff;
+}
+.body__container {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    width: 95%;
+    max-width: 1440px;
+    margin: 0 auto;
+    padding: 40px 0;
+}
+.body__instrument-panels {
+    display: flex;
+    flex-direction: column;
+    width: 228px;
+}
+.instrument-panel {
+    width: 100%;
+
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.24), 0px 0px 4px rgba(0, 0, 0, 0.12);
+
+    font-family: "Roboto";
+    font-size: 16px;
+    line-height: 24px;
+    color: rgba(0, 0, 0, 0.541327);
+
+}
+.instrument-panel .instrument,
+.instrument-panel .color {
+    display: flex;
+    justify-content: flex-start;
+    margin: 0 auto;
+    padding: 15px 0;
+    padding-left: 20px;
```

Почему бы не объеденить две строчки?
```suggestion
    padding: 15px 0 15px 20px;
```